resources/examples/simple-parsing.m4

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-
+#foo
```

Comment:
This looks like an omission.

Reply:
sorry, was forcing a change for make.  Yup need to clean this up.